eras/shelley/test-suite/cardano-ledger-shelley-test.cabal

```diff
@@ -163,6 +168,7 @@ test-suite cardano-ledger-shelley-test
       -- The 4 megabytes stack bound and 250 megabytes heap bound were
       -- determined ad-hoc.
       "-with-rtsopts=-K4m -M250m"
+      -fprint-potential-instances
```

Comment:
TIL!

Reply:
Oh no, i left this in by mistake!! Well it helped me debug, but we probably should leave it in, so good catch on you!